appng-testsupport/src/test/java/org/appng/testsupport/validation/WritingJsonValidatorTest.java

```diff
@@ -51,6 +51,15 @@ public void testValidateObject() throws Exception {
 		WritingJsonValidator.validate(p, "json/WritingJsonValidatorTest-testValidateObject.json");
 	}
 
+	@Test
+	public void testValidateObjectStrictOrder() throws Exception {
+		WritingJsonValidator.sortPropertiesAlphabetically = true;
+		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd");
+		Person p = getPerson(sdf);
+		WritingJsonValidator.validate(p, "json/WritingJsonValidatorTest-testValidateObjectAlphabeticallyOrder.json");
```

Comment:
Name of the .json should match the name of the test-method.
Suggestion: testValidateObject**Alphabetical**Order